scripts/utils/readAndWriteJson.ts

```diff
@@ -17,20 +16,34 @@ export async function writeJSON(readPath: string, writePath: string) {
   try {
     readContent = await readFile(readPath, 'utf-8');
   } catch (err) {
-    throw new Error(`Error while reading file\nError: ${err}`);
+    if (err instanceof Error) {
+      return Promise.reject(new Error(`Error while reading file\nError: ${err.message}`));
+    }
+
+    return Promise.reject(new Error('Error while reading file\nError: Unknown error'));
```

Comment:
This looks quite irrelevant to me. Can we just do `Promise.reject(err)`? Isn't this quite simple than defining error object again & again?

Reply:
Yes we can, I have wrapped it into an another error object to keep the "`Error while reading file" message as an context. 
but we can just show the error message.